src/Framework/OpenXmlBasePart.cs

```diff
@@ -81,6 +81,11 @@ internal void Load(OpenXmlPackage openXmlPackage, OpenXmlPart parent, Uri uriTar
             this._uri = uriTarget;
 
             // TODO: should we delay load?
+            if (!this.OpenXmlPackage.Package.PartExists(uriTarget))
+            {
+                throw new OpenXmlPackageException(string.Format("The part with URI target {0} does not exist in the package.", uriTarget));
+            }
```

Comment:
Thanks Thomas, the situation where a part didn't exist occurred when opening an Office document that a user uploaded. It turned out that they'd used a 3rd party OCR tool which outputs to Open XML. The file opened fine in Word, but failed to process in my tool, so it was reported to me as a defect. I wanted to collect the data about what was failing so that it could be manually fixed up if required. I was building a system which processed thousands of documents a day for translation, so I saw all sorts of dodgy files. :)